server/config/routes.rb

```diff
@@ -14,9 +14,10 @@
   # See how all your routes lay out with "rake routes".
 
   namespace :api do
-
-    resources :maps, only: [:index]
-
+    resources :users
+    resources :maps, only: [:index] # fetches geoJSON
+    resources :map_color, only: [:index] # fetches stage colors of regions
+    resources :admin
```

Comment:
confirm that admin route has been moved into its own namespace in your most recent  commit